__tests__/endpoints.test.js

```diff
@@ -432,21 +424,32 @@ describe("app", () => {
       });
     });
   });
-  describe("DELETE /api/comments/:comment_id", () => {
-    test("status 204,  no response body sent", () => {
-      return request(app).delete("/api/comments/1").expect(204);
+  describe("GET /api", () => {
+    test("status 200, responds with a JSON object of all the available endpoints on the API", () => {
+      return request(app)
+        .get("/api")
+        .expect(200)
+        .then(({ body }) => {
+          expect(body).toBeInstanceOf(Object);
+          expect(Object.keys(body).length).toBe(9);
+        });
```

Comment:
This would fail if you added more endpoints and updated your json.

You can just require in the endpoints.json file and compare it to body!